src/code_80005FD0.c

```diff
@@ -561,22 +564,25 @@ s16 get_angle_between_waypoints(Vec3f arg0, Vec3f arg1) {
     return phi_v1;
 }
 
-s32 func_80006018(f32 arg0, f32 arg1, f32 arg2, f32 arg3, f32 arg4, f32 arg5, f32 arg6, f32 arg7) {
-    f32 temp_f0;
+bool is_collide_with_vehicle(f32 vehicleX, f32 vehicleY, f32 vehicleVelocityX, f32 vehicleVelocityZ, f32 distanceX,
+                             f32 distanceY, f32 playerX, f32 playerZ) {
+    f32 velocity;
     f32 temp_f18;
 
-    temp_f0 = sqrtf((arg2 * arg2) + (arg3 * arg3));
-    if (temp_f0 < 0.01f) {
-        return 0;
+    velocity = sqrtf((vehicleVelocityX * vehicleVelocityX) + (vehicleVelocityZ * vehicleVelocityZ));
```

Comment:
dist

Reply:
it's the velocity force so velocity make more sense